mne/viz/epochs.py

```diff
@@ -891,44 +927,100 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
     fig : instance of matplotlib figure
         Figure distributing one image per channel across sensor topography.
     """
+    from matplotlib.ticker import ScalarFormatter
     from .raw import _set_psd_plot_params, _convert_psds
     fig, picks_list, titles_list, units_list, scalings_list, ax_list, \
         make_label = _set_psd_plot_params(
             epochs.info, proj, picks, ax, area_mode)
+    if spatial_colors is None:
+        spatial_colors = False if average else True
 
+    psd_list = list()
+    ylabels = list()
     for ii, (picks, title, ax) in enumerate(zip(picks_list, titles_list,
                                                 ax_list)):
-        psds, freqs = psd_multitaper(epochs, picks=picks, fmin=fmin,
-                                     fmax=fmax, tmin=tmin, tmax=tmax,
-                                     bandwidth=bandwidth, adaptive=adaptive,
-                                     low_bias=low_bias,
-                                     normalization=normalization, proj=proj,
-                                     n_jobs=n_jobs)
+        if method == 'multitaper':
+            psds, freqs = psd_multitaper(epochs, picks=picks, fmin=fmin,
+                                         fmax=fmax, tmin=tmin, tmax=tmax,
+                                         bandwidth=bandwidth,
+                                         adaptive=adaptive,
+                                         low_bias=low_bias,
+                                         normalization=normalization,
+                                         proj=proj,
+                                         n_jobs=n_jobs)
+        elif method == 'welch':
+            psds, freqs = psd_welch(epochs, picks=picks, fmin=fmin,
+                                    fmax=fmax, tmin=tmin, tmax=tmax,
+                                    n_fft=256, n_overlap=0, n_per_seg=None,
+                                    proj=proj,
+                                    n_jobs=n_jobs)
+        else:
+            raise ValueError('Only "multitaper" and "welch" are supported.')
```

Comment:
I would use `raise NotImplementedError` rather than `ValueError`. (I know we don't have a consensus in the codebase and we can find it both ways)

Reply:
@massich I'd agree with @mmagnuski here. I'd only use NotImplementedError if there is some concrete alternative that is not yet ready. ValueError is really for catching non-sensical strings. Say someone wants to do method='deep-neural-net'.

Reply:
@dengemann Speaking of which - we could have a deep neural net that classifies string arguments as non-sensical and meaningful. ;-)